munit/src/main/scala-0.21/munit/internal/Compat.scala

```diff
@@ -0,0 +1,25 @@
+package munit.internal
+
+import munit.Location
+import scala.quoted._
+import scala.reflect.Selectable.reflectiveSelectable
```

Comment:
took me a while to find, but this is the `Any => Selectable` implicit conversion, which enables the cast to a structural type in `collectionClassName` (in Dotty structural types must extend `Selectable`)